tests/sentry/demo/test_data_population.py

```diff
@@ -0,0 +1,37 @@
+from django.test import override_settings
+
+from sentry.demo.data_population import DataPopulation
+from sentry.demo.settings import DEMO_DATA_GEN_PARAMS, DEMO_DATA_QUICK_GEN_PARAMS
+from sentry.testutils import TestCase
+
+# significantly decrease event volume
+DEMO_DATA_GEN_PARAMS = DEMO_DATA_GEN_PARAMS.copy()
+DEMO_DATA_GEN_PARAMS["MAX_DAYS"] = 1
+DEMO_DATA_GEN_PARAMS["SCALE_FACTOR"] = 0.05
+
+DEMO_DATA_QUICK_GEN_PARAMS = DEMO_DATA_QUICK_GEN_PARAMS.copy()
+DEMO_DATA_QUICK_GEN_PARAMS["SCALE_FACTOR"] = 0.20
+
+
+@override_settings(
+    DEMO_MODE=True,
+    DEMO_DATA_GEN_PARAMS=DEMO_DATA_GEN_PARAMS,
+    DEMO_DATA_QUICK_GEN_PARAMS=DEMO_DATA_QUICK_GEN_PARAMS,
+)
+class DataPopulationTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.react_project = self.create_project(organization=self.organization, platform="react")
+        self.python_project = self.create_project(organization=self.organization, platform="python")
```

Comment:
@roggenkemper why do we need to create the projects? Seems like it's unused.

Reply:
remnant from past from before I moved a lot of the tests over to demo org manager. not needed here anymore.